closure/testing/test/BUILD

```diff
@@ -31,28 +31,31 @@ closure_js_library(
     deps = ["//closure/library"],
 )
 
-closure_js_test(
-    name = "arithmetic_module_test",
-    srcs = ["arithmetic_module_test.js"],
-    defs = ["--entry_point=goog:arithmetic_module_test"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
-
-closure_js_test(
-    name = "arithmetic_scope_test",
-    srcs = ["arithmetic_scope_test.js"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
+# TODO(hochhaus): Enable test when upstream local file loader fix is included
```

Comment:
https://github.com/google/closure-library/commit/6453bc5c9d526120f759266b3f4d558511a54a92 added ES6 debug loader transpilation. However, in [goog.loadFileSync_](https://github.com/google/closure-library/commit/6453bc5c9d526120f759266b3f4d558511a54a92#diff-c359099ba4bc09bc2864536c49356d52R1477) it added a bug by assuming only "200" HTTP status codes from the XHR signifies success. This works for remote files, but fails for "file://" URLs (which return status of "0" on success). The [fix](https://github.com/google/closure-library/commit/42fcead73bc36dc6586e5cbda1fe8b4416a785e6#diff-c359099ba4bc09bc2864536c49356d52R1490) just relaxes the check to `xhr.status == 0 || xhr.status == 200`.

Therefore this is an issue when running in un-compiled mode or otherwise loading off of the local file system.

What is your preferred next step?